File: RestAPIs/UserAuthApi/Models/State.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace UserAuthApi.Models
{
    [Table("state")]
    public class State
    {
        [Key]
        [Column("state_id")]
        public int StateId { get; set; }

        [Column("stateName")]
        public string StateName { get; set; }
    }
}

File: User Services/client/src/main/java/com/worksphere/dashboardapi/entity/State.java
package com.worksphere.dashboardapi.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "state")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class State {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "state_id")
    private Integer stateId;

    @Column(name = "stateName")
    private String stateName;
}
